### PR TITLE
Toolbar and headline section of portfolio

### PR DESCRIPTION
**Pull request by batoolfatima2135**
Created Toolbar and headline section the same as selected Figma template using Flexbox and fulfilling other requirements as well.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    <title>Portfolio</title>
+    <link href='https://fonts.googleapis.com/css?family=Inter' rel='stylesheet'>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <section class="main">
+        <header>
+            <div class="normal_button">
+                <img src="images/Icon - Menu.png" width="24px" height="24px" alt="Menu-Icon">
+            </div>
```
Comment (Qoosim) [inline index.html]: - [ ] It is required to wrap the hamburger inside a `nav` tag. The nav tag is for semantics reason, and the right position for convention and user experience for mobile devices. Kindly change generic div to nav tag.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+        </header>
+        <section class="headline">
+            <h2 class="greeting">Hey there.<br>I’m Alex.</h2>
```
Comment (Qoosim) [inline index.html]: - [ ] This is the main title on your page and the appropriate heading tag for it is `h1`. If you want that google (and other search engines) knows about a specific topic, in this case, you use an `h1` tag. You can check this [article](https://www.reliablesoft.net/h1-tag/) for more information.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+            <p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+               don’t hesitate to contact me.
+            </p> 
+            <section class="social_media_icons">
+                <img class="icons"  src="images/social media icons/twitter.png" alt="twitter">
+                <img class="icons" src="images/social media icons/Linkedin icon.png" alt="Linkedin">
+                <img class="icons" src="images/social media icons/medium.png" alt="medium">
+                <img class="icons" src="images/social media icons/github.png" alt="github">
+                <img class="icons" src="images/social media icons/angellist.png" alt="angellist">
+            </section>
```
Comment (Qoosim) [inline index.html]: - [ ] It is required to use `ul` to group the list of social media icons, and `li` per icon; each icon has a link (`a` tag). Kindly look into this and fix.

**Review by Qoosim — CHANGES_REQUESTED** [review]
#  *Status: Required Changes* :recycle:

Hi @batoolfatima2135, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
-  No linter errors. :heavy_check_mark: 
-  Good README file. :heavy_check_mark: 
-  Correct GitHub Flow. :heavy_check_mark: 




## Required Changes ♻️
_Check the comments under the review._

- [ ] There's a missing downward arrow on your page. Kindly add that to complete this milestone.

### Your Design
missing-part

### Figma Design
figma-part


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (batoolfatima2135) [reply index.html]: it was mentioned as h2 in the figma template so I used h2 instead of h1. But I'll change it.

File: index.html (hunk 3 above)
Comment (batoolfatima2135) [reply index.html]: do it need to provide href link as well? @Qoosim

**Review by Shedrack-Sunday — APPROVED** [review]
Hi @batoolfatima2135 

Congratulations! 🎉 Your project is complete and ready to be merged! 🚀

image


## Highlights
- Cool structure 👍 
- Moderate-looking design 👍

## Optional Suggestions 💡
While these comments with the [OPTIONAL] prefix won't prevent the approval of this pull request, I strongly recommend taking them into account as they can improve your code. Some of them were missed by the previous reviewer and addressing them will really enhance your application.
Cheers, and Happy coding!👏👏👏

If you have any questions or need clarification on anything, feel free to leave a comment in the pull request thread, and don't forget to tag me so I'll receive the notification.

As per the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project. If you feel that the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**